src/com/xmartlabs/slackbot/CommandManager.kt

```diff
@@ -138,12 +138,12 @@ object CommandManager {
         ) { _, _ ->
             """
                Hey, here there is useful information about Frogs:
```

Comment:
Just to be consistent with other commands
```suggestion
               Hey, here there is useful information about Frogs :frog: 
```

Reply:
frogs everywhere 🤣